config/secrets.yml.example

```diff
@@ -0,0 +1,11 @@
+development:
+  secret_key_base: example
+
+test:
+  secret_key_base: example
+
+production:
+  secret_key_base: example
+  AWS_ACCESS_KEY_ID: example
```

Comment:
I don't think that this will load as `ENV["AWS_ACCESS_KEY_ID"]`. We should consider solution such as https://github.com/bkeepers/dotenv